apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -568,6 +597,18 @@ def parse(element_html, data):
             )
             answer["ranking"] = search["ranking"] if search is not None else None
             answer["tag"] = search["tag"] if search is not None else None
+            if search is None:
```

Comment:
I acknowledge that this `search` variable wasn't introduced in this PR, but I don't fully understand what it is. Is there a better variable name that we can use here?

Reply:
This code goes through all blocks in the problem to find the one whose text matches a block in the student answer. 

I'll rename it to `matching_block`, that will be more useful.